docs/detections/prebuilt-rules/downloadable-packages/0-13-3/prebuilt-rule-0-13-3-web-application-suspicious-activity-post-request-declined.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[prebuilt-rule-0-13-3-web-application-suspicious-activity-post-request-declined]]
+=== Web Application Suspicious Activity: POST Request Declined
+
+A POST request to web application returned a 403 response, which indicates the web application declined to process the request because the action requested was not allowed
```

Comment:
```suggestion
A POST request to web application returned a 403 response, which indicates the web application declined to process the request because the action requested was not allowed.
```